content/en-us/projects/assets/api.md

```diff
@@ -1,195 +1,283 @@
+    </tr>
+    <tr>
+      <td>sortOrder</td>
+      <td>int</td>
+      <td>`1` = Asc<br />`2` = Desc</td>
+    </tr>
+    <tr>
+      <td>limit</td>
+      <td>int</td>
+      <td>Number of results to return, maximum 100.</td>
+    </tr>
+    <tr>
+      <td>cursor</td>
+      <td>string</td>
+      <td>Pagination cursor</td>
+    </tr>
+    <tr>
+      <td>pageNumber</td>
+      <td>int</td>
+      <td>Page number to request for</td>
+    </tr>
+    <tr>
+      <td>keyword</td>
+      <td>string</td>
+      <td>Search keyword</td>
+    </tr>
+    <tr>
+      <td>assetSubTypes</td>
+      <td>array[string]</td>
+      <td>Ad, MaterialPack, Package</td>
+    </tr>
+    <tr>
+      <td>excludeAssetSubTypes</td>
+      <td>array[string]</td>
+      <td>Ad, MaterialPack, Package</td>
+    </tr>
+    <tr>
+      <td>creatorType</td>
+      <td>int</td>
+      <td>`1` = User<br />`2` = Group</td>
+    </tr>
+    <tr>
+      <td>creatorTargetId</td>
+      <td>int64</td>
+      <td>User or Group Id to search for</td>
+    </tr>
+    <tr>
+      <td>minDuration</td>
+      <td>int64</td>
+      <td>Minimum value of the duration range for audio assets in seconds</td>
+    </tr>
+    <tr>
+      <td>maxDuration</td>
+      <td>int64</td>
+      <td>Maximum value of the duration range for audio assets in seconds</td>
+    </tr>
+    <tr>
+      <td>sortDirection</td>
+      <td>int</td>
+      <td>`0` = None<br />`1` = Ascending<br />`2` = Descending</td>
+    </tr>
+    <tr>
+      <td>artist</td>
+      <td>string</td>
+      <td>The name of the artist</td>
+    </tr>
+    <tr>
+      <td>album</td>
+      <td>string</td>
+      <td>The album you are looking for</td>
+    </tr>
+    <tr>
+      <td>audioTypes</td>
+      <td>array[int]</td>
+      <td>`0` = Music<br />`1` = SoundEffect</td>
+    </tr>
+    <tr>
+      <td>uiSortIntent</td>
+      <td>int</td>
+      <td>What sort order to rank the results by<br /><br />`1` = Relevance<br />`6` = Trending<br />`7` = AllTime<br />`8` = Top<br />`9` = Duration<br />`10` = DateCreated<br />`11` = DateModified<br />`12` = Creator<br />`13` = Name</td>
+    </tr>
+    <tr>
+      <td>includeOnlyVerifiedCreators</td>
+      <td>bool</td>
+      <td>A flag to include only results from verified creators. Verified creators are those that are ID or phone verified.</td>
+    </tr>
+    <tr>
+      <td>minPriceInCents</td>
+      <td>int64</td>
+      <td>Minimum cost in cents (only applicable to plugins)</td>
+    </tr>
+    <tr>
+      <td>maxPriceInCents</td>
+      <td>int64</td>
+      <td>Maximum cost in cents (only applicable to plugins)</td>
+    </tr>
+  </tbody>
 </table>
 
-The following URL will search for 10 items in the "Models" subcategory, sorted by most recently updated.
+For more detailed documentation regarding the toolbox-service, please refer to the updated documentation at creator store API.
```

Comment:
Which documentation is this referring to?

Reply:
I was referring to _this_ documentation that this PR added. Maybe I can reword this to 
> please refer to the updated documentation at creator store API below.